README.md

```diff
@@ -9,29 +9,29 @@ Template for developing custom Marlin packages.
 ## Setup Instructions
 
 ### Container
-All commands should be run inside the `infnpd/mucoll-ilc-framework:1.6-centos8` image.
+All commands are compatible and should be run inside the latest `gitlab-registry.cern.ch/muon-collider/muoncollider-docker/mucoll-sim:master-alma9` image (versions 2.8 and above).
 
 #### Apptainer
 ```bash
-apptainer shell --cleanenv docker://infnpd/mucoll-ilc-framework:1.6-centos8
+apptainer shell --cleanenv gitlab-registry.cern.ch/muon-collider/muoncollider-docker/mucoll-sim:master-alma9
 ```
 
 #### Shifter
 ```bash
-shifter --image infnpd/mucoll-ilc-framework:1.6-centos8 /bin/bash
+shifter --image gitlab-registry.cern.ch/muon-collider/muoncollider-docker/mucoll-sim:master-alma9 /bin/bash
 ```
 
 ### Build Instructions
-Run the following commands from inside your container. The same commands will also work with a local installation of the ILC software, with the exception of the first line.
+Run the following commands from inside your container. The same commands will also work with a local installation of the ILC and Key4Hep software, with the exception of the first line.
 ```bash
-source /opt/ilcsoft/muonc/init_ilcsoft.sh # Setup ILC software
+source /opt/setup_mucoll.sh # Setup software
 cmake -S . -B build 
 cmake --build build
 ```
 
 ### Setup Script
 The included `setup.sh` script is useful for defining all paths for the binaries built by the workspace. At the current stage, it setups the following:
-- ILC software via `init_ilcsoft.sh`
```

Comment:
via `setup_mucoll`

Reply:
this was now changed (does not show here since the comment really refers to the line after)